### ISSUE TITLE
Question on CLIP evaluation

### ISSUE THREAD

**Issue by ysharma1126** [facebookresearch/vissl#485]
[Here](https://github.com/facebookresearch/vissl/blob/main/configs/config/benchmark/linear_image_classification/kinetics700/eval_resnet_8gpu_transfer_k700_linear.yaml), I attach the config YAML for the VISSL implementation of a linear probe evaluation from the CLIP benchmark. `RandomResizedCrop` and `RandomHorizontalFlip` are specified in `DATA.TRAIN.TRANSFORMS`, resulting in a discrepancy between `DATA.TRAIN.TRANSFORMS` and `DATA.TEST.TRANSFORMS`. However, in the [linear probe evaluation](https://github.com/openai/CLIP#linear-probe-evaluation) provided in the CLIP code release, the preprocessing for train and test are the same, and are (on the whole) equivalent to what's specified in the config as `DATA.TEST.TRANSFORMS`, not `DATA.TRAIN.TRANSFORMS`. 

If this discrepancy was intentional, could the reasoning be clarified for users?

**Comment by iseessel** [comment]
CC: @QuentinDuval

**Comment by ysharma1126** [comment]
Thanks! Since we are discussing the configs, a few follow-up questions, 

1. Are the transforms always set intentionally? For example, this [config](https://github.com/facebookresearch/vissl/blob/main/configs/config/benchmark/robustness_out_of_distribution/iwilds_cam/eval_resnet_8gpu_robustness_in1k_linear.yaml) for iWILDS-Cam only uses `RandomHorizontalFlip` during training. 
2. On the topic of configs, can you confirm whether the following [config](https://github.com/facebookresearch/vissl/blob/main/configs/config/benchmark/linear_image_classification/imagenet1k/other_styles/eval_resnet_moco_8gpu_transfer_in1k_linear.yaml) replicates the linear evaluation procedure performed in the Moco paper? I find certain aspects confusing, namely some indicators (file name, sync batch norm config) imply the config was meant to use 8 GPUs, when it instead uses 1 GPU. While I understand that the number of processes shouldn't affect the experimental results (as the batch size is set correctly), such incongruencies do give me pause as a user in whether all empirical details of the reproduction are aligned with the original paper. 

Overall, I think it would be helpful if comments could be added specifying (a) the reference to what, if anything, this config should be reproducing and (b) any intentional discrepancies with the reproduction, like what was discussed above with regards to CLIP. With that being said, entirely understand if this wouldn't be worth the time cost.

**Comment by ysharma1126** [comment]
One more note on configs, the [config](https://github.com/facebookresearch/vissl/blob/main/configs/config/benchmark/object_detection/voc0712/mocoV2/rn50_transfer_voc0712_detectron2_e2e.yaml#L40) for Faster R-CNN on VOC07+12 using the Mocov2 setting differs from the [config](https://github.com/facebookresearch/moco/blob/main/detection/configs/pascal_voc_R_50_C4_24k.yaml#L16) in the Mocov2 repo w.r.t. `SOLVER.WARMUP_ITERS`. Was this intentional?
